### PR TITLE
feat(components): [tree-v2] add slot `empty`

### PR DESCRIPTION
**Pull request by zhixiaotong**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/tree-v2/src/tree.vue
Comment (warmthsea) [inline packages/components/tree-v2/src/tree.vue:1]: It looks like the formatting is not correct, can you change it in the editor?

image

Comment (zhixiaotong) [reply packages/components/tree-v2/src/tree.vue:1]: 图省事，直接点了suggestion没有验证，我本地改一下

Comment (warmthsea) [reply packages/components/tree-v2/src/tree.vue:1]: By the way, add a test case, thank you

**Review by btea — APPROVED** [review]
Thanks!

**Review by warmthsea — APPROVED** [review]

**Pull request merged**